src/styles/index.scss

```diff
@@ -1,3 +1,130 @@
-body {
+* {
+  box-sizing: border-box;
   margin: 0;
+  padding: 0;
+}
+
+.page {
+  font-family: Roboto, Manrope, sans-serif;
+  font-style: normal;
+  font-weight: 400;
+}
+
+.product {
+  display: flex;
+  flex-direction: column;
+  align-items: center;
+  padding: 32px 16px 16px;
+  width: 200px;
+  border-radius: 5px;
+  border: 1px solid #F3F3F3;
+
+  &__img {
+    width: 160px;
+    height: 134px;
+    margin-bottom: 40px;
+  }
+
+  &__title {
+    margin-bottom: 4px;
+    color: #060B35;
+    font-size: 12px;
+    font-weight: 500;
+    line-height: 18px;
+  }
+
+  &__code {
+    width: 100%;
+    margin-bottom: 16px;
+    color: #616070;
+    font-size: 10px;
+    line-height: 14px;
+  }
+
+  &__price {
+    width: 100%;
+    margin-bottom: 16px;
+    display: flex;
+    justify-content: space-between;
+
+    &-text {
+      color: #616070;
+      font-size: 12px;
+      line-height: 18px;
+    }
+    &-values {
+      font-size: 16px;
+      font-weight: 700;
+      line-height: 18px;
+    }
+  }
+
+  &__reviews {
+    margin-bottom: 24px;
+  }
+
+  &__button {
+    width: 100%;
+    display: flex;
+    align-items: center;
+    justify-content: space-around;
+    border-radius: 5px;
+    border: 1px solid transparent;
+    background-color: #00ACDC;
+    height: 40px;
+    color: #fff;
+    text-decoration: none;
+
+    font-weight: 700;
+    font-size: 14px;
+    line-height: 16px;
+    text-transform: uppercase;
+
+    &:hover {
+      background: #fff;
+      color: #00ACDC;
+      border: 1px solid #00ACDC;
+    }
+  }
+}
+
+.reviews {
+  width: 100%;
+  display: flex;
+  justify-content: space-between;
+  align-items: center;
+
+  &__text {
+    text-align: right;
+    font-size: 10px;
+    line-height: 14px;
+    white-space: nowrap;
+  }
+}
+
+.stars {
+  display: flex;
+  width: 100%;
+
+  &__star  {
+    background-image: url(../images/star.svg);
+    background-position: center;
+    background-repeat: no-repeat;
+    width: 16px;
+    height: 16px;
+    display: flex;
+    margin-right: 4px;
+  }
+
+  &--4 :nth-child(-n+4) {
```

Comment:
Consider adding styles for other modifiers to keep functionality from the previous task

Reply:
how to understand this?

Reply:
done